composer.json

```diff
@@ -64,7 +64,8 @@
     },
     "conflict": {
         "egulias/email-validator": "^2",
-        "oscarotero/html-parser": "<0.1.7"
+        "oscarotero/html-parser": "<0.1.7",
+        "symfony/process": "<4.4.30 || >=5.0.0 <5.3.7"
```

Comment:
Seems like we're better of just having <5.3.7 and prevent v4 from installing at all?

Reply:
Don't see why? 4.4.30 should run fine.

Reply:
However, it's really a non-issue. It's unlikely anyone is using 4.x of it and if they are they should be able to update it pretty easily. I'll make this change to avoid ping pong.

Reply:
Done